app/jobs/regular/create_linked_topic.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Jobs
+  class CreateLinkedTopic < ::Jobs::Base
+
+    def execute(args)
+      reference_post = Post.find_by(id: args[:post_id])
+      return unless reference_post.present?
+      parent_topic = reference_post.topic
+      return unless parent_topic.present?
+      parent_topic_id = parent_topic.id
+      parent_title = parent_topic.title
+      title_suffix_locale = I18n.t('create_linked_topic.title_suffix')
```

Comment:
```suggestion
```

Let's use a different translation for this so that we can avoid adding the parentheses in code. And, to be on the safe side, let's make it a pluralized string (see locale file).